tg/views/__init__.py

```diff
@@ -249,6 +251,35 @@ def resize(self, rows: int, cols: int, p: float = 0.5) -> None:
         self.win.resize(self.h, self.w)
         self.win.mvwin(0, self.x)
 
+    def _get_flags(self, msg_proxy: MsgProxy):
+        flags = []
+        chat = self.model.chats.chats[self.model.current_chat]
+        my_id = self.users.get_me()["id"]
```

Comment:
better to add and use method `self.users.get_my_id()` or property ``self.users.my_id``